### PR TITLE
REF: more greppable names for ujson functions

### PR DESCRIPTION
**Pull request by jbrockmendel**
Discussed with @WillAyd

### CODE REVIEW

File: pandas/_libs/src/vendored/ujson/python/ujson.c [hunk 1]
```diff
@@ -54,14 +54,9 @@ PyObject *JSONToObj(PyObject *self, PyObject *args, PyObject *kwargs);
     "encode_html_chars=True to encode < > & as unicode escape sequences."
 
 static PyMethodDef ujsonMethods[] = {
-    {"encode", (PyCFunction)objToJSON, METH_VARARGS | METH_KEYWORDS,
```
Comment (WillAyd) [inline pandas/_libs/src/vendored/ujson/python/ujson.c:57]: Didn't realize these even existed. Nice catch

**Review by WillAyd — APPROVED** [review]
Lgtm if green

**Comment by mroeschke** [comment]
```
pandas/io/json/_table_schema.py:16: error: Module "pandas._libs.json" has no attribute "ujson_loads"  [attr-defined]
pandas/io/json/_json.py:24: error: Module "pandas._libs.json" has no attribute "ujson_dumps"  [attr-defined]
pandas/io/json/_json.py:24: error: Module "pandas._libs.json" has no attribute "ujson_loads"  [attr-defined]
pandas/io/excel/_xlsxwriter.py:264: error: Module has no attribute "ujson_dumps"  [attr-defined]
pandas/io/excel/_odswriter.py:250: error: Module has no attribute "ujson_dumps"  [attr-defined]
```

**Review by mroeschke — APPROVED** [review]

**Comment by mroeschke** [comment]
Thanks @jbrockmendel

**Pull request merged**